PCI デバイスに対し割り当て済みのリソースを変更しないようにする。

Index: uefi/OvmfPkg/OvmfPkgX64.dsc
===================================================================
--- uefi.orig/OvmfPkg/OvmfPkgX64.dsc	2015-02-09 23:26:30.945305531 +0900
+++ uefi/OvmfPkg/OvmfPkgX64.dsc	2015-04-19 23:18:17.554737144 +0900
@@ -389,10 +389,7 @@
   UefiCpuPkg/CpuDxe/CpuDxe.inf
   PcAtChipsetPkg/8254TimerDxe/8254Timer.inf
   PcAtChipsetPkg/PciHostBridgeDxe/PciHostBridgeDxe.inf
-  MdeModulePkg/Bus/Pci/PciBusDxe/PciBusDxe.inf {
-    <LibraryClasses>
-      PcdLib|MdePkg/Library/DxePcdLib/DxePcdLib.inf
-  }
+  DuetPkg/PciBusNoEnumerationDxe/PciBusNoEnumeration.inf
   PcAtChipsetPkg/KbcResetDxe/Reset.inf
   MdeModulePkg/Universal/Metronome/Metronome.inf {
     <LibraryClasses>
Index: uefi/OvmfPkg/OvmfPkgX64.fdf
===================================================================
--- uefi.orig/OvmfPkg/OvmfPkgX64.fdf	2014-01-22 04:39:13.311423000 +0900
+++ uefi/OvmfPkg/OvmfPkgX64.fdf	2015-04-19 23:24:25.544626574 +0900
@@ -256,7 +256,7 @@
 INF  UefiCpuPkg/CpuDxe/CpuDxe.inf
 INF  PcAtChipsetPkg/8254TimerDxe/8254Timer.inf
 INF  PcAtChipsetPkg/PciHostBridgeDxe/PciHostBridgeDxe.inf
-INF  MdeModulePkg/Bus/Pci/PciBusDxe/PciBusDxe.inf
+INF  DuetPkg/PciBusNoEnumerationDxe/PciBusNoEnumeration.inf
 INF  PcAtChipsetPkg/KbcResetDxe/Reset.inf
 INF  MdeModulePkg/Universal/Metronome/Metronome.inf
 INF  PcAtChipsetPkg/PcatRealTimeClockRuntimeDxe/PcatRealTimeClockRuntimeDxe.inf
